.github/workflows/ci.yaml

```diff
@@ -48,6 +48,38 @@ jobs:
       - name: Test
         run: cargo test --workspace
 
+  build-msrv:
```

Comment:
Artichoke has different documented promises about its Rust toolchain support than Intaglio and other lib crates in the artichoke GitHub organization. Can we remove this additional build job?